Issue processing (#95)
Hecter94/EndlessSky-PluginArchive: res/template.txt

## Add your plugin

> Do you want your plugin here? Fork this repository, upload your plugin, and make a Pull Request. Besides that, you can post it in the discussion area, or better create a plugin submission issue.

Reviewer: ```suggestion
Do you want your plugin here? Create a Pull Request or open a new [issue](https://github.com/Hecter94/EndlessSky-PluginArchive/issues/new?assignees=&labels=&projects=&template=plugin-submission.yaml)
```
